### DISCUSSION TITLE
What is the best settings for index (recall rate) for face recognition system?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [milvus-io/milvus#36317] [Q&A and General discussion]
What is the best settings (recall rate) for face recognition system? 
What could be a starting point?

Like i care only about 10 most likely objects.

**Comment by yhmo** [comment]
I think it is case-by-case, depending on your business requirement. In most of time it is a trade-off of performance and accuracy.

**Reply by martinenkoEduard** [reply]
Is there an index with the highest recall rate for the most similar vectors.
Can you give some examples.

**Reply by yhmo** [reply]
FLAT index has the highest recall rate(100%), it is brute-force search, each vector is computed.

```
index_params = {
        "index_type": "FALT",
        "metric_type": metric_type,
        "params": {}
    }
    collection.create_index(field_name = "xxx", index_params = index_params)
```

**Comment by xiaofan-luan** [comment]
Don't think FLAT is good for large scale data set.
Check HNSW or DiskANN. It's kind like based on your data size
